tests/jobserver/models/test_outputs.py

```diff
@@ -1,31 +1,39 @@
 import pytest
+from django.conf import settings
 from django.urls import reverse
 
-from ...factories import OrgFactory, ProjectFactory, ReleaseFactory, WorkspaceFactory
+from tests.factories import ReleaseFactory
 
 
 @pytest.mark.django_db
-def test_release_creation():
-    release = ReleaseFactory(files=["file.txt"], upload_dir="workspace/release")
-
-    assert str(release.file_path("file.txt")) == "releases/workspace/release/file.txt"
-
-
-@pytest.mark.django_db
-def test_release_get_absolute_url():
-    org = OrgFactory()
-    project = ProjectFactory(org=org)
-    workspace = WorkspaceFactory(project=project)
-    release = ReleaseFactory(workspace=workspace)
+def test_get_absolute_url():
```

Comment:
Can you add the model name back into the test name please, there are lots of get_absolute_urls!

Reply:
ah, for sure